pkgs/tools/misc/collada2gltf/default.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchFromGitHub
+, lib
+, stdenv
+, cmake
+, ninja
+}:
+stdenv.mkDerivation rec {
```

Comment:
New packages should be in pkgs/by-name

Reply:
I believe that wasn't a thing when this PR was created, but fair enough. Moved it there, rebased on master, and fixed a compilation problem by switching to an older gcc version.
It seems to work again with those changes, a [sample file](https://github.com/assimp/assimp/blob/master/test/models/Collada/COLLADA.dae) converts just fine, and the created glTF file is valid.